transcom/mymove
# B-19176-INT add upload orders for SCs

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-19176)

## Summary

Services Counselors can now upload documents for customers orders.

### How to test

1. Access the application as a Services Counselor.
2. Search for a customer and create a new move.
3. Once on the move details page, click View and edit orders button
4. Because no orders documents are present, you should see the Add orders button. Clicking it should expand the upload orders section. Upload a file or files; when you finish click done and confirm the document viewer refreshes with your uploaded document(s).
5. Navigate away and return, and confirm your uploads persist, and the ability to upload files is no longer there (uploading amended orders will come later)
